### PR TITLE
Implementation of bluetooth transport in the SDL

### PR DESCRIPTION
**Pull request by nk0leg**
see detail in [SDLWIN-386](https://adc.luxoft.com/jira/browse/SDLWIN-384)

### CODE REVIEW

**Comment by nk0leg** [comment]
@malirod , @anosach-luxoft  please review

File: src/components/utils/include/utils/bluetooth_win/bluetooth_service_record.h [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  const utils::BluetoothUUID& Uuid() const {
+    return uuid_;
+  }
+
+  /**
+   * @brief Check if equil stored UUID and service_uuid.
+   *
+   * @return True if service_uuid and stored UUID equil.
+   */
+  bool IsUuidEquil(const std::string& service_uuid) const {
```
Comment (anosach-luxoft) [inline src/components/utils/include/utils/bluetooth_win/bluetooth_service_record.h]: @nk0leg `equil` looks like mistake

File: src/components/utils/src/socket_win.cc [hunk 2]
```diff
@@ -99,6 +99,8 @@ class utils::TcpSocketConnection::Impl {
 
   bool Connect(const HostAddress& address, const uint16_t port);
 
+  bool Connect(const SOCKET bth_soket);
```
Comment (anosach-luxoft) [inline src/components/utils/src/socket_win.cc]: @nk0leg do not use `bth` prefix here. This method could be used not only for BT

File: src/components/utils/include/utils/bluetooth_win/bluetooth_service_record.h (hunk 1 above)
Comment (nk0leg) [inline src/components/utils/include/utils/bluetooth_win/bluetooth_service_record.h]: see f045cd6

File: src/components/utils/src/socket_win.cc (hunk 2 above)
Comment (nk0leg) [inline src/components/utils/src/socket_win.cc]: see f045cd6

**Comment by anosach-luxoft** [comment]
Reviewed

**Comment by nk0leg** [comment]
@malirod  please look 7e6d24d

**Comment by nk0leg** [comment]
see #169

**Pull request closed without merging**